cql/ManageRareAbnormality.cql

```diff
@@ -824,6 +852,96 @@ define CytologyInterpretedAsHsil:
       return aC in "HSIL"
   )  
 
+define CervixRemovalDateIsAfterMostRecentAisBiopsy:
+  (CervixRemovalDate is not null) and 
+  CervixRemovalDate after Collate.MostRecentAisBiopsy.date
+    
+define MostRecentAisBiopsy:
+  Collate.MostRecentAisBiopsy
+
+define MostRecentTreatmentAfterAis:
+  MostRecentTreatment P
+  where CCF.ProcedureDate(P) occurs after Collate.MostRecentAisBiopsy.date
+
+define HasTreatmentAfterAis:
+  MostRecentTreatmentAfterAis is not null  
+
+// I.6 Management of AIS over 50: Adoption of Society of Gynecologic Oncology Recommendations
+// Severity of AIS should recommend treatment until hysterectomy is performed.
+define RecommendationForAisHistologyResults:
+  case
+    when (
+      AgeInYears() > 50 and
+      Collate.MostRecentAisBiopsy is not null and
+      not CervixRemovalDateIsAfterMostRecentAisBiopsy and
```

Comment:
This is an interesting requirement to put in here because UPMC has told us they will not be returning hysterectomies with dates, therefore `CervixRemovalDateIsAfterMostRecentAisBiopsy` will always evaluate to false here, even if the patient did have a hysterectomy. That being said, I think this is technically correct, and I don't actually think this would casue any issues, since if a patient had a hysterectomy without a date, they would be getting a recommendation from the Special Population section, which would be evaluated first.

You can go ahead and resolve this if you think we are good to proceed as is, but I just wanted to bring this up.